package.json

```diff
@@ -1542,7 +1542,7 @@
           "type": "string",
           "default": "#ffffff",
           "markdownDeprecationMessage": "**Deprecated**: Please use `#latex-workshop.view.pdf.color.light.backgroundColor#` and `#latex-workshop.view.pdf.color.dark.backgroundColor#` instead.",
-          "deprecationMessage": "Deprecated: Please use `#latex-workshop.view.pdf.color.light.backgroundColor#` and `#latex-workshop.view.pdf.color.dark.backgroundColor#` instead."
```

Comment:
The syntax `#config.variable#` is actually used be vs code to create a link to the new configuration variable. See https://code.visualstudio.com/api/references/contribution-points paragraph _Linking to settings_.

Reply:
The latest doc changed the scope as follows:

> `#target.setting.id#`. This will work in markdownDescription, markdownEnumDescriptions, and markdownDeprecationMessage.

Therefore, only deprecateMessages are changed.

Reply:
My bad. I did not pay attention to this change.